### DISCUSSION TITLE
EVAL-Mode; Bridge Interfaces on VMware workstation

### DISCUSSION THREAD

**Discussion by ehsannt10** [Security-Onion-Solutions/securityonion#5555] [Unsupported Versions] [answered]
Hello,

my VM has three virtual NICs and I used nmcli to create a bridge interface and added two virtual interfaces to BR0. 
ens34 and ens36 are slave interfaces for br0 and br0 is on promisc mode.
Can I use br0 as a monitor interface between my VMs?
I am using securityonion-2.3.70-WAZUH.iso image.
![image](https://user-images.githubusercontent.com/25203981/133727335-384063d9-a3c4-4e8a-93c1-0245ec3a30a0.png)

**Comment by dougburks** [comment, marked as answer]
It sounds like you're trying to run Security Onion inline, but it is not designed for that. You might be able to make this work, but we don't recommend or support it.

**Reply by ehsannt10** [reply]
thanks for your reply. It seems it need many modifications for running inline mode.
